ai_ta_backend/vector_database.py

```diff
@@ -906,12 +906,12 @@ def split_and_upload(self, texts: List[str], metadatas: List[Dict[str, Any]]):
           "contexts": contexts_for_supa,
       }
 
-      self.supabase_client.table(
+      response = self.supabase_client.table(

```

Comment:
Capturing and using Supabase response object for logging to document maps. Looks like below - 

```
Response from supabase:  data=[{'id': 2, 'created_at': '2024-02-02T14:47:45.104334-08:00', 'course_name': 'NCSADelta', 'doc_map_id': '413fa075-c749-45d2-8d6f-27a294d22fdc', 'convo_map_id': None}] count=None
```